src/components/families/FamilyDetailsSidebar.tsx

```diff
@@ -80,6 +87,10 @@ const FamilyDetailsSidebar = ({
               {family.parent.information[parentField.id]}
             </Typography>
           ))}
+          <Typography variant="body2" className={classes.heading}>
+            Notes
+          </Typography>
+          <TextField value={family.notes} fullWidth variant="filled" />
```

Comment:
it would be good to have forms in `<form>` tags and ensure each input is linked to a label for accessibility purposes! more deets in the [textfield docs](https://material-ui.com/components/text-fields/#accessibility)

[this example](https://github.com/mui-org/material-ui/issues/13840#issuecomment-445037184) might be helpful for styling the label - if we don't want the shrinking animation, we can set `shrink: true` on the input label ([docs](https://material-ui.com/components/text-fields/#shrink))